src/System.Console/tests/NegativeTesting.cs

```diff
@@ -9,17 +9,33 @@
 public class NegativeTesting
 {
     [Fact]
+    [SkipOnTargetFramework(TargetFrameworkMonikers.Uap)] // In appcontainer, the stream cannot be opened: there is no Console
```

Comment:
@danmosemsft With your recent added PlatformDetection property you should now be able to change the condition to `[ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWinRT))]`. That brings two befinits: 
1. If uapaot runs inside an AppContainer the test will be skipped and won't throw
2. If uap runs outside of an AppContainer (for whatever reason) the test will be executed and won't skipped.